src/styles/base/_override.typography.scss

```diff
@@ -15,8 +15,17 @@
 .ds-text-heading--lg,
 .ds-text-heading--md,
 .ds-text-heading--sm {
+  font-family: Montserrat;
```

Comment:
Can we use a variable here instead of the font itself?

Reply:
This is complete.